lmnet/executor/profile_model.py

```diff
@@ -147,21 +175,46 @@ def helper(node, level):
         is_quant_kernel = all([layer not in node.name for layer in unquant_layers]) and "kernel" == \
                           node.name.split("/")[-1]
         bits = bit if is_quant_kernel else 32
+        node_name = "total" if level == 0 else node.name
         res.append(
-            [level, node.name, node.total_parameters, node.total_parameters * 32, node.total_parameters * bits])
+            [level, node_name, node.total_parameters, node.total_parameters * 32, node.total_parameters * bits])
         idx = len(res) - 1
+
+        node_param_dict = {
+            'name': node_name,
+            'parameters': node.total_parameters,
+            'size': node.total_parameters * 32,
+            'quant_size': node.total_parameters * bits,
```

Comment:
Just question. Conv layer can be quantized and parameter size is `bits`.
Other layer type (for example, fully connected) will not be quantized.
This size calculation assume all of our parameters are Conv?

Reply:
@iizukak Thank you for your question!
No, if a layer is not a quantizable kernel, `quant_size` will be a same value of `size`.
`quant_size` is determined as following steps.

1. check node_name is kernel or not, see [is_quant_kernel](https://github.com/blue-oil/blueoil/pull/691/files#diff-925afadc461a809c6ff64ea888587efaR175)
2. set `bits = bit if is_quant_kernel else 32`
3. calc `'quant_size': node.total_parameters * bits,`

This is the previous implementation, but it is difficult to understand as you mentioned.

Ummm. OK, I am planing to change these logics as follows.
```
node_params = node.total_parameters
node_size = node_params * 32
node_quant_size = (node_params * bit) if is_quant_kernel else None
```
This plan is to change quant_size is `None` if it cannot be quantized.
What do you think?

Reply:
> This plan is to change quant_size is None if it cannot be quantized.

I tried and implement that, but it is a little complicated to use `None` and `int` in the same dictionary key's value and this implementation changes the current Markdown file format. So, I would not like to use `None` to follow the current format. Then, `node_quant_size` will be `(node_params * bit) if is_quant_kernel else 
 node_size`

Reply:
@iizukak 
I changed to use named values for node parameters so this becomes better to understand, I think. Please re-review this! 👍